src/mount/mount_unmount.rs

```diff
@@ -49,6 +49,8 @@ pub fn mount<Source: path::Arg, Target: path::Arg, Fs: path::Arg, Data: path::Ar
 ///
 /// [Linux]: https://man7.org/linux/man-pages/man2/mount.2.html
 #[inline]
+#[deprecated(note = "This API was added in error, use the expressive mount APIs instead.")]
+#[doc(hidden)]
```

Comment:
Since we're nearing a 1.0 release, we can just delete this function instead of deprecating it.